src/infrastructure/storage/note.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Note data interface
+ */
+interface NoteData {
```

Comment:
maybe we can get rid of such structures and return a Note Entity itself?

Reply:
done